apps/schedule/feeds.py

```diff
@@ -46,12 +49,146 @@ def schedule_json(year):
     return Response(json.dumps(schedule), mimetype="application/json")
 
 
+@schedule.route("/schedule/schedule-<int:year>.json")  # TODO validate url with upstream
```

Comment:
Maybe we can just use `/schedule/<int:year>.json` and merge this pull request after this years EMF?